src/Helper/ConfigHelper.php

```diff
@@ -88,28 +132,52 @@ public function getReferrerIds()
         return $referrerIds;
     }
 
-	public function getSmartCheck()
-	{
-		return $this->config->get('EkomiFeedback.smart_check');
-	}
-
-	public function getProductIdentifier()
-	{
-		return $this->config->get('EkomiFeedback.product_identifier');
-	}
-
-	public function getExcludeProducts()
-	{
-		return $this->config->get('EkomiFeedback.exclude_products');
-	}
-
-	public function getShowPrcWidget()
-	{
-		return $this->config->get('EkomiFeedback.show_prc_widget');
-	}
-
-	public function getPrcWidgetToken()
-	{
-		return $this->config->get('EkomiFeedback.prc_widget_token');
-	}
+    /**
+     * Gets Smart Check from plugin configurations.
+     *
+     * @return mixed
```

Comment:
please specify the return type(s) instead of mixed

Reply:
fixed